docker/db/scripts/00_init_test_db.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+##############################################################
+# Initialize a second database used for integration testing  #
+##############################################################
+MYSQL_DATABASE_TEST="${MYSQL_DATABASE}_test"
+
+echo "Creating database ${MYSQL_DATABASE_TEST}..."
+mysql -uroot -p"${MYSQL_ROOT_PASSWORD}" -e "create database ${MYSQL_DATABASE_TEST};"
+
+echo "Importing fixtures into database ${MYSQL_DATABASE_TEST}..."
+mysql -uroot -p"${MYSQL_ROOT_PASSWORD}" ${MYSQL_DATABASE_TEST} < ../test_db.sql
```

Comment:
This is the local path, from the docker-compose.yml volume, so thats the only place that didnt need to change :)